.github/workflows/kind.yml

```diff
@@ -88,46 +99,69 @@ jobs:
       env:
         deployOnHub: ${{ matrix.deployOnHub }}
         KIND_VERSION: ${{ matrix.kind }}
+        HOSTED: ${{ matrix.hosted }}
```

Comment:
Nit: I don't think this env is used outside of the workflow, so there's not a real benefit to defining an env here rather than using `${{ matrix.hosted }}` directly? (Same with the E2E step)

Reply:
I prefer to not use github expressions inside the `run`. When I do that, I worry that I haven't written it correctly, and I'll get a weird syntax error either in GH or the shell. Keeping the expressions in the `env` block means the `run` block is a valid, standalone script.